### ISSUE TITLE
Sharing Log Files seamlessly within the ownCloud Android App

### ISSUE THREAD

**Issue by dpapac** [owncloud/android#4156]
### Actual behaviour

When attempting to share log files with our own application, it does not appear as a selectable option in the sharing menu.

### Expected behaviour

Our own application should be listed among the available apps for sharing when trying to share log files.

**Comment by michaelstingl** [comment]
@dpapac what exact version? Post a screenshot?

**Comment by JuancaG05** [comment]
This is a explicit restriction specified in the current code:

```
// Show dialog, without the own app
val packagesToExclude = arrayOf<String>(this.packageName)
```

So the best thing to do is create a new brandable parameter (`allow_sharing_with_own_app`?) which states if we can share with the own app or not, keeping `false` as default value to keep current behaviour. Just in case there is a historical reason for that restriction to be there.

**Comment by michaelstingl** [comment]
No problem to show own app from my POV

**Comment by jesmrec** [comment]
Done. To be released in 4.2.
